### ISSUE TITLE
🐛 [firestore-bigquery-export] Fields with the type of timestamp are imported as a string instead of a JSON struct (re-open).

### ISSUE THREAD

**Issue by RyosukeNishi** [firebase/extensions#1849]
This is re-opened from https://github.com/firebase/extensions/issues/1813

### [READ] Step 1: Are you in the right place?

Issues filed here should be about bugs for a **specific extension in this repository**.
If you have a general question, need help debugging, or fall into some
other category use one of these other channels:

- For general technical questions, post a question on [StackOverflow](http://stackoverflow.com/)
  with the firebase tag.
- For general Firebase discussion, use the [firebase-talk](https://groups.google.com/forum/#!forum/firebase-talk)
  google group.
- To file a bug against the Firebase Extensions platform, or for an issue affecting multiple extensions, please reach out to
  [Firebase support](https://firebase.google.com/support/troubleshooter/contact/) directly.

### [REQUIRED] Step 2: Describe your configuration

- Extension name: **firestore-bigquery-export**
- Extension version:
  - Extension itself:
    - 0.1.41
  - Import script:
    - 0.1.19
- Configuration values (redact info where appropriate):
<img width="644" alt="スクリーンショット 2023-11-28 15 42 39" src="https://github.com/firebase/extensions/assets/46443906/cd11e273-d534-40da-b177-1d11071b9d8c">
<img width="647" alt="スクリーンショット 2023-11-28 15 42 46" src="https://github.com/firebase/extensions/assets/46443906/e2e832db-aed2-41ac-8248-3c1ef6a0c7a8">

#### Steps to reproduce:
```
$ fs-bq-import-collection --version
{"severity":"WARNING","message":"Warning, FIREBASE_CONFIG and GCLOUD_PROJECT environment variables are missing. Initializing firebase-admin will fail"}
0.1.19

$ fs-bq-import-collection \
  --non-interactive \
  --project $MY_PROJECT_NAME \
  --source-collection-path users/{user_id}/bookmarks \
  --query-collection-group true \
  --dataset firestore_export \
  --table-name-prefix bookmarks \
  --batch-size 300 \
  --dataset-location us \
  --multi-threaded true \
  --use-new-snapshot-query-syntax true \
  --use-emulator false
```

##### Expected result
<img width="485" alt="スクリーンショット 2023-11-21 13 10 25" src="https://github.com/firebase/extensions/assets/46443906/640c2577-d06d-499d-a625-8100b739b066">

##### Actual result
<img width="585" alt="スクリーンショット 2023-11-21 13 08 20" src="https://github.com/firebase/extensions/assets/46443906/c7c1f5dc-763e-45b0-b0cc-bf040e03dbd7">

**Comment by cabljac** [comment]
It seems the latest version didn't publish on npm. I will publish it now. Thanks!

**Comment by cabljac** [comment]
A new version is now published, 0.1.20, which should fix this. Let me know if it still occurs.

**Comment by RyosukeNishi** [comment]
I confirmed that the issue is resolved. Thanks!
